tox-travis.ini

```diff
@@ -0,0 +1,220 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
```

Comment:
Should we rename this file to be not tied to "travis"? maybe tox-ci.ini?

Reply:
I am not sure that a different CI system would be able to use this file as is. So at this point, I am only confident it will work with Travis CI.